index.Rmd

```diff
@@ -0,0 +1,461 @@
+    title = "Number of Extreme Hot Days Per Year, by Location" ,
+    caption = "Data from www.noaa.gov")
+
+
+
+```
+
+
+
+The station in SD was missing measures for 2019, but had the highest number of cold days in 2020 and 2021.  The OH station had the next highest number of hot days.  
+
+The SD and OK stations need to be checked based on 1 extreme day,  The OH station does not have the highest or lowest temperatures, but there is enough variation between hot and cold that there could be more need for batteries there.  
+
+
+## Timing During the Year
+
+```{r, warning=FALSE, message=FALSE}
+weather %>% 
+  group_by(Location, Year, Quarter ) %>% 
+  summarise( num_hot_days = sum(as.numeric(hot_flag)),
+             num_cold_days = sum(as.numeric(cold_flag))) %>% 
+  ungroup() %>% 
+  ggplot( aes ( x = Quarter,
+                 y=  num_cold_days,
+                color = Location,
+                fill = Location) ) +
+  geom_col(position = "dodge", na.rm = TRUE) +
+  theme_minimal() +
+  theme(legend.position = "bottom" ,
+        legend.direction = "vertical",
+        axis.text = element_text(face = "italic") ) +
+  labs(
+        y = "Number of Extreme Days" ,
+    title = "Number of Extreme Cold Days Per Quarter, by Location" ,
+    caption = "Data from www.noaa.gov")
+
+
+
+```
+
+
+
+Only SD had a significant number of cold days in Q2.  All stations had extreme temperatures in Q1, and most have at least a trend for cold days in Q4. 
+
+
+```{r, warning=FALSE, message=FALSE}
+weather %>% 
+  group_by(Location, Year, Quarter ) %>% 
+  summarise( num_hot_days = sum(as.numeric(hot_flag)),
+             num_cold_days = sum(as.numeric(cold_flag))) %>% 
+  ungroup() %>% 
+  ggplot( aes ( x = Quarter,
+                 y=  num_hot_days,
+                color = Location,
+                fill = Location) ) +
+  geom_col(position = "dodge", na.rm = TRUE) +
+  theme_minimal() +
+  theme(legend.position = "bottom" ,
+        legend.direction = "vertical",
+        axis.text = element_text(face = "italic") ) +
+  labs(
+        y = "Number of Extreme Days" ,
+    title = "Number of Extreme Hot Days Per Quarter, by Location" ,
+    caption = "Data from www.noaa.gov")
+
+```
+
+
+
+Q3 is where most stations see their hottest days.  the OK and PA stations also had hot days in Q2. This again shows that not all the areas have the same weather timing, at least not for the temperatures which can damage car batteries. 
+
+
+# Correlation
+
+## Between area and temperature
+
+
+```{r, warning=FALSE, error=FALSE, message=FALSE}
+weather %>% 
+  group_by(STATION,LATITUDE, Year) %>% 
+  summarise( num_hot_days = sum(as.numeric(hot_flag)),
+             num_cold_days = sum(as.numeric(cold_flag))) %>% 
+  pivot_longer( starts_with("num_"),
+               names_to = "type" ) %>% 
+  ungroup() %>% 
+  ggplot( aes ( x = LATITUDE,
+                y = value,
+                color = type
+               )) +
+  geom_point() +
+  geom_smooth(na.rm = TRUE) +
```

Comment:
I'm not sure what this `na.rm = TRUE` is doing here

Reply:
We didn't cover this in class directly, but I'd probably change the calculation method for `geom_smooth` to being linear. You can do that as follows:

`geom_smooth(method = "lm")`

Output:
![image](https://user-images.githubusercontent.com/16887262/167767744-f06db5de-8e0a-4dd0-bcfe-4f86031ad238.png)

Reply:
The reason being that I don't think a Loess smoothing is appropriate with so few data points. To be fair, a linear model isn't really appropriate either...but I feel a bit better about it in this particularly circumstance.